src/parse/lexer.rs

```diff
@@ -27,6 +27,7 @@ pub fn str_to_keyword(word: &str) -> Option<Tok> {
         "import" => Some(Tok::Import),
         "let" => Some(Tok::Let),
         "opaque" => Some(Tok::Opaque),
+        "inline" => Some(Tok::Inline),
```

Comment:
I was wondering, is `inlined type` better than `inline type`?

Reply:
To my ears "inlined type" sounds like it's already inlined in some way while "inline type" sounds like it will be inlined wherever it's used. But neither is obviously wrong or right. It's up to you.

Reply:
Sounds good, let's keep it with "inline" then. Thank you